mantidproject/mslice
# Save to ASCII or Matlab formats

Adds ability to save cuts in other file formats.
Adds dropdown to `WorkspaceManager` save button to select a file format. If saving a single file, the format can also be changed in the save dialog.
These new save options are also accessible using the save button in the plot window (although only cuts can currently be saved in the new formats, support for slices / `MDEventWorkspace` will be added in another PR).
Cuts saved as `ASCII` files can now be loaded.

**To test:**
- Save different workspace types as `.txt` or `.mat` files.
- Save a cut in the new formats using the save button in the plot window.
- Load a cut from a `.txt` file and check it plots correctly.

Fixes #251
